docs/cleanup.md

```diff
@@ -0,0 +1,35 @@
+# Cleanup
+
+To cleanup TopoLVM, please follow these steps.
+
+**Caution: The cleanup procedure cannot be reverted. Think twice whether you can really cleanup TopoLVM.**
+
+1. Delete all the snapshots and PVCs related to TopoLVM.
+2. If the scheduler extender is enabled, edit the kube-scheduler's config file and remove TopoLVM's settings. It may be your help to read [the configuration guide](https://github.com/topolvm/topolvm/blob/0ccaaa31779c2fd1c51f657a488e39d40afe5b46/deploy/README.md#configure-kube-scheduler).
```

Comment:
This permanent link references the old content if we will edit the configuration guide of the kube-scheduler. It may be better to use the url of main branch although the link will be broken in the future.

Reply:
I fixed it.